### PR TITLE
Updated stability to have a log output

### PR DESCRIPTION
**Pull request by ssarthak01**

### CODE REVIEW

File: StabilityLog.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Equilibrium solution when E,T=0 [0.0, 0.0, 0.5555555555555556]
+Stability analysis: asympotically stable
+
+Equilibrium solution when E=0 [0.0, -127899200824.95488, 0.5555555555555556]
+Stability analysis: asympotically stable
+
+Equilibrium solution when T=0 [32039.87184051264, 0.0, 0.5555555555555556]
+Stability analysis: asympotically stable
+
```
Comment (SidhantPuntambekar) [inline StabilityLog.txt]: Can you specify what these equilibrium solutions mean? Are they in terms of T, E, M or something else? Also, do these solutions match up with what is in the paper (maybe clarify how we are getting these numbers through documentation)?

**Review by SidhantPuntambekar — COMMENTED** [review]
Could we add some documentation explaining the log and stability analysis? See comment on the log specifically.

**Review by SidhantPuntambekar — APPROVED** [review]
Yep this looks good. @stefan-wass please review as well.

**Review by stefan-wass — APPROVED** [review]
Looks good

**Pull request merged**